ipfx/aibs_data_set.py

```diff
@@ -90,7 +92,7 @@ def extract_sweep_meta_data(self):
             sweep_record["stimulus_code_ext"] = stim_code_ext
             sweep_record["stimulus_code"] = stim_code
 
-            if self.ontology:
+            if self.validate_stim:
```

Comment:
I think you want to move the validation into `EphysDataSet.get_stimulus_name`.  The way this is, the "name" column will only be populated if you are validating.  

Expected behavior to me:
1) `validate_stim = True`: populate "name" column of stim table but raise an exception if name is not in the ontology
2) `validate_stim = False`: populate "name" column of stim table but raise a warning if name is not in the ontology, leave "name" as None if name is not found.